### PR TITLE
Add aria-labelledby for input field

### LINKED ISSUE #8080: IgxDatePicker accessibility issue
## Description  
Using the following code for igx-datepicker
```
 <igx-date-picker
            [label]="'Date of Birth'"
            name="dateOfBirth"
            [(ngModel)]="generalInfoObj.retailMemberPii.dateOfBirth"
          ></igx-date-picker>
```
Unfortunately generated code fails accessibility testing.

 * igniteui-angular version: 10.1.2

## Expected result  
Input fields should always have a description that is explicitly associated with the field to make sure that users of assistive technologies will also know what the field is for.
If the input field has a visible description indicating the purpose of the field, this description should be explicitly associated to the input field either as an HTML label or using the WAI-ARIA attribute 'aria-labelledby'.
If it is not possible to add a visible description, either add a mouseover text ('title' attribute) to the input field or create an invisible label using the WAI-ARIA attribute 'aria-label'.

### PR DESCRIPTION
**Pull request by IvayloG**
Closes #8080 

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [x] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [x] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts [hunk 1]
```diff
@@ -32,7 +33,13 @@ import {
     isDateInRanges
 } from '../calendar/public_api';
 import { IgxIconModule } from '../icon/public_api';
-import { IgxInputGroupModule, IgxInputDirective, IgxInputGroupComponent, IgxInputState } from '../input-group/public_api';
+import {
+    IgxInputGroupModule,
+    IgxInputDirective,
+    IgxInputGroupComponent,
+    IgxInputState,
+    IgxLabelDirective
```
Comment (wnvko) [inline projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts:41]: you have this imported again on row 71

File: projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts [hunk 2]
```diff
@@ -833,6 +849,11 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
         return this._inputDirective || this._inputDirectiveUserTemplates.first || null;
     }
 
+    /** @hidden @internal */
+    public get labelDirective(): IgxLabelDirective {
```
Comment (wnvko) [inline projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts:853]: there is field with same name, `labelDirective` on line 671.

File: projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts [hunk 3]
```diff
@@ -688,6 +698,12 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
     @ContentChildren(IgxInputDirective, { descendants: true })
     private _inputDirectiveUserTemplates: QueryList<IgxInputDirective>;
 
+    @ViewChild(IgxLabelDirective)
+    private _labelDirectiveDefault: IgxLabelDirective;
```
Comment (wnvko) [inline projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts:702]: IMO `_labelDirective` without `Default` is ok like name. This is the way we named `_inputGroup` and `_inputDirective`

**Review by wnvko — CHANGES_REQUESTED** [review]

File: projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts (hunk 2 above)
Comment (IvayloG) [reply projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts:846]: resolve merge master

**Review by wnvko — APPROVED** [review]

**Pull request merged**